src/App.css

```diff
@@ -1,5 +1,216 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@200;300;400;500;600;700;800;900&display=swap");
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Montserrat, sans-serif;
+}
+
+body {
+  background: #f5f6fa;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 1.438rem 7.5rem 1.688rem 7.5rem;
+}
+
+ul.navbar {
+  width: 50%;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+ul > h1 {
+  color: #0290ff;
+  font-size: 1.875rem;
+}
+
+ul.navbar > li {
+  margin: 10px 0;
+  color: #000;
+}
+
+ul.navbar li a {
+  color: #888;
+}
+
+.profile {
+  justify-content: center;
+  align-items: center;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  padding: 12px;
+}
+
+.bookH3 {
+  font-size: 1.25rem;
+  font-weight: bold;
+  color: #888;
+  margin: 1.438rem 7.5rem 0.5rem 7.5rem;
+  padding-top: 1.438rem;
+  border-top: 2px solid #888;
+}
+
+form {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 4fr));
+  width: 80%;
+  justify-content: space-between;
+  margin: 1.438rem 7.5rem 1.4rem 7.5rem;
+  gap: 34px;
+}
+
+input,
+select {
+  width: 100%;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+}
+
+.addBtn {
+  width: 150px;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  cursor: pointer;
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-family: "Roboto Slab", sans-serif;
+}
+
+.statusBtn {
+  display: block;
+  width: 150px;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  cursor: pointer;
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-family: "Roboto Slab", sans-serif;
+  margin: 100px auto;
+}
+
+.bookContainer {
+  background-color: white;
+  margin: 1.438rem 7.5rem 1.688rem 7.5rem;
+  border: 1px solid #888;
+  padding: 1.438rem 1.5rem;
+  border-radius: 10px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Your design is almost exceptional @Donmark2k, But there are some minor details to check. As you can see from the Zeplin design the border for the `bookContainer` is not visible black, instead, it has a very thin box shadow which sounds like a border. This is to make sure you have the exact design clone.
### Your design. 
![image](https://user-images.githubusercontent.com/69077061/222795684-8402e1d9-379b-46cc-bab2-5dcdf856d540.png)
### Zeplin design
![image](https://user-images.githubusercontent.com/69077061/222795748-62b67ebb-93f2-464c-9ba2-595d0788348c.png)